### ISSUE TITLE
Create screen taping jig?

### ISSUE THREAD

**Issue by scottbez1** [scottbez1/smartknob#21]
It's a little tricky to ensure the LCD is perfectly centered relative to the screen PCB (and therefore the rest of the mechanics and knob window) when permanently mounting it with VHB, so it might be nice to have a small 3d-printable jig to help with alignment.

It would hold the screen PCB (with alignment pin to ensure correct rotation) and have walls that align the LCD. Might be tricky since the LCD will be rotating down via its ribbon cable rather than simply translating straight down, but could be nice to get better overall alignment considering how permanent VHB is.

(If anyone tackles this, note that the existing LCD model in the F360 file is only a rough depiction based on a handful of dimensions from the datasheet. It would probably be best to put the LCD on a flatbed scanner to ensure dimensional accuracy)

**Comment by scottbez1** [comment]
When building one, I realized this would be trickier than originally thought, since the screen is adhered only once all the display wires have been connected, so it can only be done with the screen PCB fully mounted to the rest of the electronics. That would make it difficult to create an alignment jig keyed to the PCB.

What might be feasible is to create keying indents on the sides of the screen PCB, and then make a 3d printed 2-piece jig that aligns on either side of the screen PCB using those indents for alignment, and has walls for aligning the LCD as you flip it down.
